src/main/java/Listmanager.java

```diff
@@ -0,0 +1,133 @@
+import java.util.Scanner;
+
+public class Todolist {
+
+    String Icon = "T";
+    private static final int MAX_LIST_LENGTH = 100;
+    static String[] todolist = new String[MAX_LIST_LENGTH];
+    static boolean[] isDoneList = new boolean[MAX_LIST_LENGTH];
+
+    static int ListCount= 0;
+    static Scanner in = new Scanner(System.in);
+
+    public static void AddToList(){
+        String Task= in.nextLine();
+        todolist[ListCount] = Task;
+        isDoneList[ListCount]= false;
+        ListCount++;
+        return;
+    }
+
+
+
+    static void HandleList(){
+        PrintList();
+        String command ="";
+
+        while(!command.equals("quit")){
+            command = in.nextLine();
+            switch (command){
+            case"add":
```

Comment:
Inconsistency in whether there is a space after case.
```suggestion
            case "add":
```